nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2314,10 +2314,10 @@ async fn image_global_view(
 /// Fetch a global image by id
 #[endpoint {
     method = GET,
-    path = "/by-id/global-images/{id}",
-    tags = ["images:global"],
+    path = "/system/by-id/images/{id}",
```

Comment:
I don't feel too strongly either way but I kind of thought this would be `/by-id/system/images`. Is that bizarre?

Reply:
It's not bizarre, that certainly is a plausible approach. The reason I went with `/system` being first is because it just treats `/by-id/` in this context just like any other route that's being executed outside the context of the silo. `/by-id/system/` on the other hand is sort of a special case. The docs would be something like: _"All non-silo scoped endpoints grouped under `/system` unless you want to do an ID lookup then it's `/by-id/system`"_ vs _"All non-silo scoped endpoints are grouped under `system`"_.

Reply:
My reasoning is that it simplifies the algorithm for constructing a `by-id` path. If it's `/by-id/system/images`, you can simply say "take whatever the path is and prefix it with `/by-id`", and that would be true for all (I think) resources. It looks a bit weird, but to me both versions look equally weird, so it being one trivial universal rule sells it for me.

Reply:
But yeah I see the other side too. "Why is this system route not prefixed with `/system`?" Ugh!

Reply:
I don't feel strongly about this but `/system/by-id` makes a little more sense to me because I think of `/` and `/system` as basically two different namespaces, each with a top-level `by-id` resource.  (Yeah, it's a little weird to think of it that way because one is contained in the other.  But that's my mental model.)

Reply:
Yeah, I’m fine with it. We’ll try to make it easy to see in the API docs.